Containerfile

```diff
@@ -28,18 +23,18 @@ COPY deploy/ deploy/
 COPY hack/ hack/
 COPY internal/ internal/
 
-RUN make build
+RUN GOOS=$TARGETOS GOARCH=$TARGETARCH go build -mod=vendor --ldflags "-s -w" -a -o bin/kubesan cmd/main.go
 
 # CentOS Stream 9 doesn't provide package nbd
 # FROM quay.io/centos/centos:stream9
-FROM quay.io/fedora/fedora:40
+FROM --platform=$TARGETPLATFORM fedora:latest
```

Comment:
Why allow the Fedora version to change externally? This may cause breakage when a new fedora:latest is released. Locking the version number minimizes the risk of build/CI failures caused by external version changes.

Reply:
No arguments from me, I'll set this back to 40

Reply:
As long as you're looking at this, would you also look at: https://gitlab.com/kubesan/kubesan/-/merge_requests/100/diffs?commit_id=89a370dcde47c8cb2b421245344c147b46984478 and see where I am trying to add 'nbd-client -i' into KubeSAN?  My initial approach was to add a new deps/Containerfile and stash the just-build nbd-client into quay.io/kubesan/nbd-client-i so that not all developers have to repeat the steps - but if we are going to parameterize the build to run multi-arch, then nbd-client-i also needs to be built multi-arch for the same set of architectures.